spec/requests/submission_spec.rb

```diff
@@ -14,7 +14,7 @@
     expect(NotificationService).to receive(:post_submission_event).once
   end
 
-  it 'Completes 3' do
+  it 'Completes a submission from Tokyo' do
```

Comment:
this test description was changed recently by mistake, I just changed it to correct text. not directly related to the task.